.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules
+.vscode
```

Comment:
It's best to add this to your personal global .gitignore. http://egorsmirnov.me/2015/05/04/global-gitignore-file.html